README.md

```diff
@@ -597,6 +597,8 @@ deploy`) in the affected deployment.
 
 # 6. Branch flow & development process
 
+## Note: The section below regarding branch management is out-of-date.
```

Comment:
I wouldn't use a heading style for the note. IIRC, GH markdown does not support note boxes. If that's the case, I would use just bold.

```suggestion
## Note: This section should be considered a draft. It describes a future extension to the current branching flow.
```

Reply:
went with bold && used suggested message change.